### PR TITLE
DBR-213 - General form messages

### PR DESCRIPTION
**Pull request by andresmoschini**
Hey team! I am touching SCSS hehe!

The idea is unifying general form errors and field errors, and in both of them support basic HTML: p, span, strong, em, links, etc. 

By the moment, li is not being supported, because I do not need it yet.

Could you review it?

Thanks @GusBaamonde for your help!

@mvillaseco, @cbernat, @IgnacioRodrigues, @damianmuti 

You can review it in (in the mockups the font is wrong, in build 224 it is fixed)

* https://cdn.fromdoppler.com/doppler-ui-library/build224/signup.html
    
    image

* https://cdn.fromdoppler.com/doppler-ui-library/build224/login.html

    image

### CODE REVIEW

File: assets/scss/core/_base.scss [hunk 1]
```diff
@@ -236,6 +236,11 @@ input[type="tel"] {
   display: none;
 }
 
+.form-message {
+  margin-bottom: $dp-spaces--lv2;
+  font-family: $dp-font-family;
```
Comment (andresmoschini) [inline assets/scss/core/_base.scss]: Hey @GusBaamonde, we did not see it, but without this change, span is rendered with serif font. Do you think that it is fine? should apply it in another place?

Comment (andresmoschini) [reply assets/scss/core/_base.scss]: @GusBaamonde commented me that @santiagopaolucci said that we should use `$dp-font-family-proximanova` in all public pages.

I will check if "validation errors" is the only missing, in that case, I will fix it, if not, I will create an issue.

**Comment by andresmoschini** [comment]
You can review it in 

* https://cdn.fromdoppler.com/doppler-ui-library/build220/signup.html
    
    image

* https://cdn.fromdoppler.com/doppler-ui-library/build220/login.html

    image

**Comment by andresmoschini** [comment]
> ⚠️ No mergear! Falta la manito en el link!

Listo, faltaba el href

File: assets/scss/core/_base.scss (hunk 1 above)
Comment (andresmoschini) [reply assets/scss/core/_base.scss]: The fix is ready! Resolving the conversation...

**Comment by cbernat** [comment]
Great job! All in order 👍

**Pull request merged**